app/build.gradle

```diff
@@ -34,8 +34,10 @@ android {
 dependencies {
     implementation 'androidx.core:core-ktx:1.8.0'
     implementation 'androidx.appcompat:appcompat:1.4.2'
-    implementation 'com.google.android.material:material:1.6.1'
+    implementation 'com.google.android.material:material:1.7.0-alpha03'
```

Comment:
Needed for https://github.com/material-components/material-components-android/issues/2726

Reply:
Is this gonna be a problem when building with AOSP?

Reply:
I think the correct thing to do is importing all the libraries to match whatever gradle uses
That makes our lives much easier